### ISSUE TITLE
Way to show metrics?

### ISSUE THREAD

**Issue by hannahkane** [OpenMatt/lovetrumpshate#8]
Can we think of ways to show metrics or otherwise demonstrate successes? It would be motivating to see how many other people have given a gift, or shared socially.

**Comment by OpenMatt** [comment]
Demonstrating success / momentum might break down into:
1) data (amount of $ given, etc.) and 
2) stories (tweets & shares from people saying they're getting on board. 

(1) seems hard, whereas (2) seems easy. (1) is hard because it seems a lot easier to just pass donations through to the individal sites donate pages, versus through some central workflow. So I'm not sure of the right way to show metrics? Aside from easy stuff like shares. 

Any thoughts @hannahkane ? I'm not really a campaigner -- no idea what I'm doing. :)

**Comment by hannahkane** [comment]
Would love thoughts from @valianttry on this topic. Maybe a different hashtag to distinguish gifts from other tweets?

**Comment by OpenMatt** [comment]
maybe as simple as:  #lovetrumpshate #give ?

**Comment by hannahkane** [comment]
Yeah, that could work. Then, maybe adding that stream to the page?
